cypress/integration/axisnow-encrypted-epub/navigations.ts

```diff
@@ -0,0 +1,114 @@
+import { IFRAME_SELECTOR } from '../../support/constants';
+
+describe('navigating an EPUB page', () => {
+  beforeEach(() => {
+    cy.loadPage('/axisnow-encrypted');
+  });
+
+  it('should contain a link to return to the homepage', () => {
+    cy.findByRole('link', { name: 'Return to Homepage' }).should(
+      'have.prop',
+      'href',
+      `${Cypress.config().baseUrl}/`
+    );
+  });
+
+  it('Paginated mode & TOC: should update page content after clicking on TOC link', () => {
+    cy.intercept(
+      'GET',
+      '/samples/dickens-axisnow/encrypted/OEBPS/vol-001-chapter-002.xhtml'
+    ).as('theMailChapter');
+    cy.findByRole('button', { name: 'Settings' }).click();
+    cy.findByText('Scrolling').click();
+    cy.findByRole('radio', { name: 'Paginated' }).should('not.be.checked');
+    cy.findByRole('radio', { name: 'Scrolling' }).should('be.checked');
+    cy.findByRole('button', { name: 'Table of Contents' }).click();
+    cy.findByRole('menuitem', { name: 'The Mail' }).click();
+    cy.wait('@theMailChapter', { timeout: 10000 }).then((interception) => {
+      assert.isNotNull(
+        interception?.response?.body,
+        'chapter API call has data'
+      );
+    });
+
+    cy.wait(3000);
+
+    cy.getIframeHead(IFRAME_SELECTOR).contains('title', 'The Mail');
+    cy.findByRole('button', { name: 'Next Page' }).click();
+    // FIXME: Turning pages is not working as expected in paginated mode.
+    // Update this test as we fix https://jira.nypl.org/browse/SFR-1364
```

Comment:
Writing this test is where I noticed that pagination and displaying chapters correctly isn't working. @Toxiapo agreed do look into this.